tests/helpers/linux/test_net.py

```diff
@@ -27,3 +32,11 @@ def test_netdev_get_by_name(self):
         for index, name in socket.if_nameindex():
             netdev = netdev_get_by_name(self.prog, name)
             self.assertEqual(netdev.ifindex, index)
+
+    def test_for_each_netns(self):
+        self.assertTrue(
+            any(
+                netns.value_() == self.prog["init_net"].address_
+                for netns in for_each_netns(self.prog)
+            )
+        )
```

Comment:
This can be written more succinctly as:

```suggestion
        self.assertIn(self.prog["init_net"].address_of_(), for_each_netns(self.prog))
```

Reply:
Wow, that's so much cleaner, thanks